### PR TITLE
OCPQE-14828: kubevirt-hypershift-workflow on baremetal

### PR DESCRIPTION
**Pull request by LiangquanLi930**
add `kubevirt-hypershift-workflow` for qe

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
@jianlinliu @liangxia @kuiwang02 Could you help to review this PR? Thanks

File: ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/provision/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/provision/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision-chain.yaml:2]: If this chain is only used for the next integration, why not move the steps to `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift-provision`?

Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/provision/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision-chain.yaml:2]: this workflow `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt` still uses this step

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/provision/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision-chain.yaml:2]: Is `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt` workflow will be covered in ci or some other place is using it?

Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/provision/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision-chain.yaml:2]: In the future, will need it, ecosystem, and cnv team

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/provision/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-provision-chain.yaml:2]: okay.

File: ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-workflow.yaml:2]: I saw `cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift` set `DEVSCRIPTS_CONFIG` env, this workflow does not need those settings?

Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/ipv4/kubevirt/cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-workflow.yaml:2]: yep, Sorry, i forget it, update done

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**